src/msw/listctrl.cpp

```diff
@@ -103,9 +103,20 @@ namespace
 inline bool
 wxGetListCtrlSubItemRect(HWND hwnd, int item, int subitem, int flags, RECT& rect)
 {
+    int ret;
     rect.top = subitem;
     rect.left = flags;
-    return ::SendMessage(hwnd, LVM_GETSUBITEMRECT, item, (LPARAM)&rect) != 0;
+    ret = ::SendMessage(hwnd, LVM_GETSUBITEMRECT, item, (LPARAM)&rect);
+    if( flags == LVIR_LABEL && ret != 0 )
```

Comment:
With the addition of this block of code, the comment before this function is not correct any more, it's not just a replacement for `ListView_GetSubItemRect()` any more.

Either the comment should be updated or this needs to be done in a different function and IMO the latter is clearly preferable as `LVIR_LABEL` seems to be only used in `wxListCtrl::GetSubItemRect()`, so this code should go there.

Reply:
@vadz,
The LVIR_LABEL is a parameter that is passed into this function as a flags variable from the caller. And the caller is actually translates the wx constant to the Windows constant in order to retrieve the bounding box of the appropriate area.

Now, I agree that the comment needs to be updated, but I don't think that creating extra function is worth it.

Any objection?

Reply:
Have you actually looked at the caller? If you had, you would have noticed that the only place where `LVIR_LABEL` is used is in `wxListCtrl::GetSubItemRect()` and that this is the only place where it makes sense to make this adjustment.

Reply:
@vadz ,
I hope that my last commit is what you had in mind...